data/views/article-teaser.php

```diff
@@ -2,19 +2,25 @@
     <?php if ($image != "") : ?>
     <div class="col-12">
         <h2><a href="<?=$url?>"><?=$title?></a></h2>
+                <h3><?=$subtitle?></h3>
+
         <small><?=$createdate?> | <?=$author?></small>
     </div>
     <div class="col-4">
         <?=$image?>
     </div>
     <div class="col-8">
         <?=$teasertext?>
+        <a href="<?=$url?>" class="btn">Weiterlesen …</a>
```

Comment:
Ich bin hier nicht im Thema, aber: Mehrsprachigkeit bedenken und Button besser nicht fest benamen?

Reply:
die mehrsprachigkeit wird in der NewsManager.php berücksichtigt – und die views sind nur beispiele/ausgangsmaterial. aber ideen sind willkommen ;–)

Reply:
Ah, okay.